app/src/src/components/common/PickImage.jsx

```diff
@@ -0,0 +1,37 @@
+import * as ImagePicker from 'expo-image-picker';
+
+// select an image from the camera roll and change the image
+// doesn't actually save the image yet...
```

Comment:
Put `# todo: ` in front so it's easier to find in the future. Also, can you make a ticket to implement it so we don't forget? Thanks!

Reply:
I think there's already a ticket for it: https://github.com/Team-uMigrate/umigrate/issues/410